Octokit/Models/Request/CollaboratorRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class CollaboratorRequest
+    {
+        public CollaboratorRequest()
+        {
+            Permission = Permission.Push;
```

Comment:
I guess the API default only really applies if we dont pass the field at all?
I dont really have a problem with having to explicitly specify the permission when constructing the object, but if you feel strongly enough about it feel free to keep the default constructor, just make sure to add an XmlDoc comment to state that it defaults to Push access